contracts/RenPool.sol

```diff
@@ -94,15 +94,15 @@ contract RenPool {
 	modifier onlyOwnerNodeOperator() {
 		require (
 			msg.sender == owner || msg.sender == nodeOperator,
-			"Caller is not owner/nodeOperator"
+			"RenPool: Caller is not owner nor node operator"
```

Comment:
Nice! The rationale behind adding the error source, _e.g._, `RenPool`, is to make explicit which contract is throwing the error.